specification/automation/resource-manager/Microsoft.Automation/stable/2018-06-30/python2package.json

```diff
@@ -0,0 +1,733 @@
+        },
+        "version": {
+          "type": "string",
+          "description": "Gets or sets the version of the module."
+        },
+        "sizeInBytes": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the size in bytes of the module."
+        },
+        "activityCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets or sets the activity count of the module."
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Gets or sets the provisioning state of the module.",
+          "enum": [
+            "Created",
+            "Creating",
+            "StartingImportModuleRunbook",
+            "RunningImportModuleRunbook",
+            "ContentRetrieved",
+            "ContentDownloaded",
+            "ContentValidated",
+            "ConnectionTypeImported",
+            "ContentStored",
+            "ModuleDataStored",
+            "ActivitiesStored",
+            "ModuleImportRunbookComplete",
+            "Succeeded",
+            "Failed",
+            "Cancelled",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "ModuleProvisioningState",
+            "modelAsString": false
+          }
+        },
+        "contentLink": {
+          "$ref": "#/definitions/ContentLink",
+          "description": "Gets or sets the contentLink of the module."
+        },
+        "error": {
+          "$ref": "#/definitions/ModuleErrorInfo",
+          "description": "Gets or sets the error info of the module."
+        },
+        "creationTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the creation time.",
+          "x-nullable": false
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the last modified time.",
+          "x-nullable": false
+        },
+        "description": {
+          "type": "string",
+          "description": "Gets or sets the description."
+        },
+        "isComposite": {
+          "type": "boolean",
+          "description": "Gets or sets type of module, if its composite or not."
+        }
+      },
+      "description": "Definition of the module property type."
+    },
+    "ModuleUpdateProperties": {
+      "properties": {
+        "contentLink": {
+          "$ref": "#/definitions/ContentLink",
+          "description": "Gets or sets the module content link."
+        }
+      },
+      "description": "The parameters supplied to the update properties."
+    },
+    "ModuleUpdateParameters": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ModuleUpdateProperties",
+          "x-ms-client-flatten": true,
+          "description": "Gets or sets the module update properties."
+        },
+        "name": {
```

Comment:
Is name of the resource really updatable?

Reply:
no, only the tag is updatable. Left in there for consistency with PowerShell modules.

Reply:
Consistency at the expense of something which doesnt work is not recommended. Please fix.